guides/keras_cv/generate_with_stable_diffusion.py

```diff
@@ -0,0 +1,294 @@
+"""
+Title: Generate images using KerasCV's StableDiffusion's at unprecedented speeds
+Author: [divamgupta](https://github.com/divamgupta), [fchollet](https://github.com/fchollet), [lukewood](https://lukewood.xyz), [ianstenbit](https://github.com/ianstenbit), ... and others if anyone wants to contribute
+Date created: 2022/09/24
+Last modified: 2022/09/24
+Description:
+"""
+
+"""
+## Overview
+
+In this guide, we will show how to generate novel images based on a text prompt using
+the KerasCV implementation of [stability.ai's](https://stability.ai/) image to text
+model,
+[StableDiffusion](https://github.com/CompVis/stable-diffusion).
+
+StableDiffusion is a powerful, open-source text to image generation model.  While there
+exist numerous open source implementations that allow you to easily create images from
+textual prompts, KerasCV's offers a few distinct advantages.
+These include [XLA compilation](https://www.tensorflow.org/xla) and
+[mixed precision computation](https://www.tensorflow.org/guide/mixed_precision).
+
+In this guide, we will explore KerasCV's StableDiffusion implementation, show how to use
+these powerful performance boosts, and explore the performance benefits
+that they offer.
+
+To get started, lets install a few dependencies and sort out some imports:
+"""
+
+"""shell
+pip install ez-timer luketils
+"""
+
+import keras_cv
+from luketils import visualization
```

Comment:
Happy to add them to KerasCV - I think for now we should do it as a follow-up though, because I actually think visualization is a place KerasCV can shine and we should put a lot of work into designing the APIs there carefully.

I feel "good" about the APIs in Luketils but not "great"